pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -687,6 +687,14 @@ func GetRecordingRules(namespace string) []KubevirtRecordingRule {
 			MType:       prometheusv1.MetricTypeCounter,
 			Description: "The total number of requests to deprecated KubeVirt APIs.",
 		},
+		{
```

Comment:
@machadovilaca  Sorry but I am not following the deprecation logic here, you actually state about deprecation of `kubevirt_vmi_cpu_affinity` but you also remove that metric, and make it a rule instead? can you please elaborate on the strategy?